commons/helpers/pods_helper.py

```diff
@@ -581,18 +581,22 @@ def apply_k8s_deployment(self, file_path: str):
                                 read_lines=True,exc=False)
         return True, resp
 
-    def select_random_pod_container(self,pod_prefix: str,
-                                    container_prefix: str):
+    def select_random_pod_container(self, pod_prefix: str,
+                                    container_prefix: str, **kwargs):
         """
         Select random pod and container for the given pods and container prefix
-        :param master_node: Logical Node object for master node.
         :param pod_prefix: Pod prefix
         :param container_prefix: Container Prefix
+        :keyword bool specific_pod: True for retrieving containers from specific pod
         return pod_selected,container_selected
         """
-        pod_list = self.get_all_pods(pod_prefix=pod_prefix)
-        sys_random = random.SystemRandom()
-        pod_selected = pod_list[sys_random.randint(0, len(pod_list) - 1)]
+        specific_pod = kwargs.get("specific_pod", False)
+        if specific_pod:
+            pod_selected = pod_prefix
```

Comment:
how will get container list will work for pod_prefix only?

Reply:
get container list requires pod name only. 

So in case of just pod_prefix, random pod will be selected from pod list and later that random pod is sent to get container function. In second case, if user sends specific_pod == True, instead of random pod, that specific pod is sent to get container list function.